broker/lib/fabrik/Fabrik.js

```diff
@@ -1,73 +1,47 @@
 'use strict';
 
-const assert = require('assert');
-const Promise = require('bluebird');
-const config = require('../../../common/config');
-const catalog = require('../../../common/models').catalog;
 const Agent = require('../../../data-access-layer/service-agent');
-const VirtualHostAgent = require('./VirtualHostAgent');
-const DirectorManager = require('./DirectorManager');
-const VirtualHostManager = require('./VirtualHostManager');
 const DBManager = require('./DBManager');
 const OobBackupManager = require('./OobBackupManager');
-const BasePlatformManager = require('../../../platform-managers/BasePlatformManager');
+const catalog = require('../../../common/models/catalog');
+const DirectorService = require('../../../operators/bosh-operator/DirectorService');
+const DockerService = require('../../../operators/docker-operator/DockerService');
+const VirtualHostService = require('../../../operators/virtualhost-operator/VirtualHostService');
 const CONST = require('../../../common/constants');
-const DockerManager = config.enable_swarm_manager ? require('./DockerManager') : undefined;
+const config = require('../../../common/config');
```

Comment:
so this config to enable/disable swarm manager will no longer be required , correct?

If yes, can you remove it from settings.yml as well to keep things clean.

@amitmalav : i guess this was done to handle ABAP use case right ?

Reply:
Yes, earlier it was by default loading DockerManager at import.
Sorry I missed the mention earlier.